### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by hafizul94**
Added few hiragana to the greetings and few sample sentences.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,15 +95,15 @@
 											<tr><!--For some reason, chg in padding works only inline with !important-->
 												<td style="padding-top: 0 !important;
 												padding-bottom: 0 !important;" colspan="3" class="hiddenRow">
-													<div id="greet1" class="collapse"><em>&nbsp;&nbsp;Sample sentence</em></div>
+													<div id="greet1" class="collapse"><em>&nbsp;&nbsp;Ohayo gozaimasu, sensei Azizol.</em></div>
```
Comment (Zyten) [inline index.html]: can pls modify to Azizol sensei?

File: index.html [hunk 2]
```diff
@@ -143,9 +143,8 @@
 											<!-- End sentence toggle-->
 																					
 											<tr class="accordion-toggle" data-toggle="collapse" data-target="#greet5">
-												<td data-th="Word">genki desuka</td>
+												<td data-th="Word">ogenki desuka</td>
 												<td data-th="Meaning">How are you?</td>
-												<td data-th="Hiragana"></td>
```
Comment (Zyten) [inline index.html]: terdelete?

**Comment by Zyten** [comment]
yo, can edit the 2 lines i commented and send pull request again? tqvm

File: index.html [hunk 3]
```diff
@@ -178,7 +177,7 @@
 																														
 											<tr class="accordion-toggle" data-toggle="collapse" data-target="#greet8">
 												<td data-th="Word">anata no namae wa nandesuka</td>
-												<td data-th="Meaning">What's your name??</td>
+												<td data-th="Meaning">What's your name?</td>
```
Comment (hafizul94) [inline index.html]: Removed one extra '?'

**Pull request closed without merging**